symplyphysics/symbols/basic.py

```diff
@@ -64,6 +69,9 @@
 radius_of_curvature = SymbolNew("r", units.length)
 """
 **Radius of curvature** is the inverse of curvature and is equal to the distance to the center of curvature.
+
+..
+    TODO are this and distance_to_axis interchangeable?
```

Comment:
Guess so :)

Reply:
I guess I would leave `radius_of_curvature` untouched and rather replace the symbol in the laws I was concerned about.